### PR TITLE
Preparation for 4.0.10 (stable-4.0)

### PR DESCRIPTION
**Pull request by simondeziel**
This PR should have a commit list very similar to that of #13643

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -92,6 +92,11 @@ ifneq "$(LXD_OFFLINE)" ""
 	exit 1
 endif
 	go get -t -v -d -u ./...
+
+	# Manual fixups
+	go get github.com/mdlayher/ndp@v0.8.0
+	go get github.com/canonical/dqlite@HEAD
```
Comment (markylaing) [inline Makefile:98]: I realise this is a cherry-pick but I'm a bit confused by it. As far as I can tell `github.com/canonical/dqlite` doesn't contain any Go code? Also, what does `@HEAD` do here?

File: doc/metadata.yaml
Comment (ru-fu) [inline doc/metadata.yaml:1]: This file isn't (or at least shouldn't :wink: ) be used anymore, so it could be deleted.

**Review by ru-fu — APPROVED** [review]
Looks good to me.
Great job getting the docs to build again! :+1:

File: Makefile (hunk 1 above)
Comment (simondeziel) [reply Makefile]: Indeed, that was not doing anything useful/effective so I dropped it. Thanks!

**Review by tomponline — APPROVED** [review]
Thanks!

**Pull request merged**